lsp-test/src/Language/LSP/Test/Session.hs

```diff
@@ -513,7 +513,7 @@ logMsg t msg = do
   shouldColor <- asks $ logColor . config
   liftIO $ when shouldLog $ do
     when shouldColor $ setSGR [SetColor Foreground Dull color]
-    putStrLn $ arrow ++ showPretty msg
+    B.putStrLn $ arrow <> encodePretty msg
```

Comment:
It seems pointless to convert pretty-printed json bytestring to String just to print it.
Wouldn't directly printing ByteString by faster?

Reply:
I have no idea, but seems sensible